### ISSUE TITLE
[Bug]: The diff view shows unexpected green dot lines

### ISSUE THREAD

**Issue by Kotsuha** [pragma-git/pragma-git#7]
### What happened?

![image](https://github.com/pragma-git/pragma-git/assets/10438710/7ac2809f-aef2-4b71-b21a-a11017cc7743)

### To reproduce

Create a new empty repository

Create a `manifest.json` file
```
{
  "dependencies": {
    "com.cysharp.unitask": "https://github.com/Cysharp/UniTask.git?path=src/UniTask/Assets/Plugins/UniTask"
  }
}
```

Commit `manifest.json`

Modify `manifest.json`
```
{
  "dependencies": {
    "com.coffee.ui-particle": "https://github.com/mob-sakai/ParticleEffectForUGUI.git",
    "com.cysharp.unitask": "https://github.com/Cysharp/UniTask.git?path=src/UniTask/Assets/Plugins/UniTask"
  }
}
```

Go to diff view and there will be green lines

### Expected behavior

It's a little weird to see green lines here.
If it wants to support typo check, maybe add a toggle.
Or don't check typo at all.

### System Info and Screen shots

Information
Pragma-git version:
0.9.11

Latest available version:
0.9.11 [[Download page]](https://github.com/pragma-git/pragma-git/releases/latest)

nwjs version:
0.83.0(sdk)

Platform:
win32

Git version:
git version 2.43.0.windows.1

Settings folder:
C:\Users\usr2\.Pragma-git

Log file:
C:\Users\usr2\.Pragma-git\.tmp\pragma-git-log.txt

Your system's path as seen by Pragma-git :
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\bin;
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\libnvvp;
C:\Windows\system32;
C:\Windows;
C:\Windows\System32\Wbem;
C:\Windows\System32\WindowsPowerShell\v1.0\;
C:\Windows\System32\OpenSSH\;
C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;
C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;
C:\Program Files\7-Zip;
C:\Program Files\TortoiseHg\;
C:\Program Files\Perforce\;
C:\Program Files\dotnet\;
C:\Program Files (x86)\GtkSharp\2.12\bin;
C:\Program Files\Nintendo\Nintendo Dev Interface;
C:\ProgramData\chocolatey\bin;
C:\Program Files\nvm;
C:\Program Files\nodejs;
C:\Program Files\Python3.10.6\Scripts\;
C:\Program Files\Python3.10.6\;
C:\PortableApps (CLI)\jdk-17.0.7+7\bin;
C:\PortableApps\CommonFiles\OpenJDKJRE64\bin;
C:\Program Files (x86)\Common Files\Oracle\Java\javapath;
C:\Program Files\Oracle\VirtualBox;
C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;
C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;
C:\Program Files\Go\bin;
C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin;
C:\Program Files\TortoiseGit\bin;
C:\Program Files\Git\cmd;
C:\Users\usr2\AppData\Local\Microsoft\WindowsApps;
C:\Program Files\Git\bin;
C:\PortableApps (CLI)\ffmpeg;
C:\PortableApps (CLI)\sox;
C:\PortableApps (CLI)\yt-dlp;
C:\PortableApps (CLI)\livedl;
C:\PortableApps (CLI)\JetBrains.ReSharper.CommandLineTools;
C:\PortableApps (CLI)\FART;
C:\PortableApps (CLI)\jq;
C:\PortableApps (CLI)\Neovim\bin;
C:\PortableApps (CLI)\bit;
C:\PortableApps\cygwin-portable\cygwin\bin;
C:\PortableApps (Not Stealth)\Arsenal-Image-Mounter;
C:\Program Files (x86)\Vim\vim90;
C:\PortableApps\MSYS2Portable\App\msys64\usr\bin;
C:\PortableApps\MSYS2Portable\App\msys64\mingw64\bin;
C:\PortableApps\VSCode-Portable\VSCode\Code.exe;
C:\Users\usr2\go\bin;
C:\Users\usr2\AppData\Roaming\Python\Python310\Scripts;
C:\Users\usrpub\.pyenv\pyenv-win\bin;
C:\Users\usrpub\.pyenv\pyenv-win\shims;
C:\Users\usr2\.dotnet\tools;
C:\Users\usr2\AppData\Local\JetBrains\Toolbox\scripts

License:
[MIT license](chrome-extension://gboiinhcjcmbpcfnoccodedfpeecdgoa/settings.html#).

**Comment by pragma-git** [comment]
Thanks for the issue report.

A green line signifies additions (and would appear in the left pane).  Similarly, a red line on the right pane would show deletions.  I will describe this in the help for the diff viewer.

Please let me know if this explains the described behavior, or if I have missed something in what you have observed

**Comment by Kotsuha** [comment]
Thanks for clarifying.

Most compare tools I use mark the whole line 4 as added, whereas Pragma mark

`offee.ui-particle": "https://github.com/mob-sakai/ParticleEffectForUGUI.git",
    "com.c`

as added.

**Comment by pragma-git** [comment]
The issue with _partial  lines_ are fixed  in commit 0b656b9d0a2e04f8bf2c259abf2b13b3cee2cadc
